examples/slsa-demo/chains-config-map.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: chains-config
+  namespace: tekton-chains
+  labels:
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-chains
+data:
+  artifacts.pipelinerun.format: in-toto
+  artifacts.pipelinerun.storage: grafeas
+  artifacts.pipelinerun.signer: kms
+  signers.kms.kmsref: gcpkms://projects/jagathprakash-test/locations/us/keyRings/pj-keyring/cryptoKeys/pj-key/cryptoKeyVersions/1
```

Comment:
Remove reference to cloud projects - include instructions for bootstrapping instead.